tools/osbuilder/rootfs-builder/README.md

```diff
@@ -193,3 +193,7 @@ To add additional packages, use one of the following methods:
 
 Once the rootfs directory is created, you can add and remove files as
 needed. Changes affect the files included in the final guest image.
+
+#### Confidential containers support
+
+When building the rootfs for confidential containers the `SKOPEO_UMOCI` parameter can be passed in and set to `yes`. This results in the `skopeo` and `umoci` packages being built into the rootfs. It also adds the signature verification proof of concept files. For more info on these, see [the documentation](signed-container-artifacts/README.md).
```

Comment:
- Please fold this long line.
- Also it might be clearer to mention `SKOPEO_UMOCI="yes"` rather than spelling it all out.

Reply:
Updated and folded the line